packages/dds/sequence/package.json

```diff
@@ -109,8 +109,8 @@
 		"test:memory": "mocha --config src/test/memory/.mocharc.cjs",
 		"test:memory-profiling:report": "mocha --config src/test/memory/.mocharc.cjs",
 		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
-		"test:mocha:cjs": "mocha --ignore \"dist/test/memory/**/*\" --recursive \"dist/test/*.spec.*js\" --exit",
-		"test:mocha:esm": "mocha --ignore \"lib/test/memory/**/*\" --recursive \"lib/test/*.spec.*js\" --exit",
+		"test:mocha:cjs": "mocha --ignore \"dist/test/memory/**/*\" --recursive \"dist/test/**/*.spec.*js\" --exit",
+		"test:mocha:esm": "mocha --ignore \"lib/test/memory/**/*\" --recursive \"lib/test/**/*.spec.*js\" --exit",
```

Comment:
should this be updated everywhere, as i see this pattern used in other packages too. i think all packages should support sub-dirs regardless of if they use them today or not

Reply:
agree. I haven't noticed that change in other PRs I've reviewed, but then 4 reviewers missed it in this prior one.

Reply:
I did a survey of where tests may have been dropped. The only other case appeared to be in dds/matrix, which is now also fixed here.